TCAT/Controllers/RouteOptionsViewController.swift

```diff
@@ -327,36 +331,50 @@ class RouteOptionsViewController: UIViewController {
         routeResults.reloadData()
     }
     
-    private func getDelay(tripId: String, stopId: String) -> Future<Response<Int?>> {
-        return networking(Endpoint.getDelay(tripID: tripId, stopID: stopId)).decode()
+    private func getAllDelays(trips: [Trip]) -> Future<Response<[Delay]>> {
+        return networking(Endpoint.getAllDelays(trips: trips)).decode()
     }
-
-    @objc func updateLiveTracking(sender: Timer) {
-        // For each route in each route array inside of the 'routes' array,
-        // retrieve its delay. Use index of route to save delay for route to
-        // JSON file.
+    
+    @objc func updateAllRoutesLiveTracking(sender: Timer) {
+        var trips: [Trip] = []
```

Comment:
Is this supposed to be related to the `trip` class variable?

Reply:
I moved out the for loop so now I use the global trips variable as opposed to the local one!